triage/os_version_linux.go

```diff
@@ -0,0 +1,27 @@
+package triage
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+func printOSVersion() {
+	file, err := os.Open("/etc/os-release")
+	if err != nil {
+		fmt.Println("Error reading OS release:", err)
+		return
+	}
+	defer file.Close()
+
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		line := scanner.Text()
+		if strings.HasPrefix(line, "PRETTY_NAME=") {
```

Comment:
Per the freedesktop spec, [`PRETTY_NAME`](https://www.freedesktop.org/software/systemd/man/latest/os-release.html#PRETTY_NAME=) may include a version number, so [`NAME`](https://www.freedesktop.org/software/systemd/man/latest/os-release.html#NAME=) would be preferable since we explicitly pull `VERSION`.